geostreams/gltg
# Create state portal options

Linted index file as well, I am going to create another branch to standardise linting in the project

## To test:
Just run the website and check if state portal options is opening in both the full size header and small screen header and the functionality is as expected
